app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/SelectedSessionParameter.kt

```diff
@@ -0,0 +1,32 @@
+package nerd.tuxmobil.fahrplan.congress.details
+
+/**
+ * Payload of the observable property in the [SessionDetailsViewModel]
+ * which is observed by the [SessionDetailsFragment].
+ */
+data class SelectedSessionParameter(
+
+    // Details content
+    val hasDateUtc: Boolean,
+    val formattedZonedDateTime: String,
+    val roomName: String,
+    val sessionId: String,
+    val title: String,
+    val subtitle: String,
+    val speakerNames: String,
+    val formattedAbstract: String,
+    val abstract: String,
+    val formattedDescription: String,
+    val description: String,
+    val hasLinks: Boolean,
+    val formattedLinks: String,
+    val hasWikiLinks: Boolean,
+    val sessionLink: String,
```

Comment:
Maybe a bit more grouping (separated by newlines) might increase readability.

Reply:
:heavy_check_mark: Done in d6583a5dd41c1ab01cb82979b18e28a5d1e52546.